.travis.yml

```
@@ -0,0 +1,10 @@
+language: javascript
+addons: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)